src/lib/OpenEXRCore/internal_dwa_compressor.h

```diff
@@ -251,20 +251,11 @@ DwaCompressor_compress (DwaCompressor* me)
     uint8_t*  outDataPtr;
     uint8_t*  inDataPtr;
 
-    // Starting with 2, we write the channel
+    // Starting with fileversion 2, we write the channel
     // classification rules into the file
-    if (fileVersion < 2)
-    {
-        me->_channelRules = sLegacyChannelRules;
-        me->_channelRuleCount =
-            sizeof (sLegacyChannelRules) / sizeof (Classifier);
```

Comment:
I see you are deleting this code because fileVersion is hard coded to 2 above. It would make sense to also delete the `sLegacyChnanelRules` array as well, right?

Reply:
sLegacyChannelRules is still used elsewhere so we cannot delete it.